lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -189,7 +191,120 @@ function _signup_resource_index($user, $users, $campaigns, $count, $page, $compe
     'runs' => $runs,
   ];
 
-  return (new SignupTransformer)->index($parameters);
+  if (variable_get('rogue_collection', FALSE)) {
+    if ($user) {
+      $northstar_user_id = dosomething_user_get_northstar_id($user);
```

Comment:
ah interesting! I tried `/api/v1/signups?users=1704953,1709171&campaigns=1360` and it didn't work. I will work on this - thanks for flagging!